geotools/build.sbt

```diff
@@ -0,0 +1,30 @@
+import Dependencies._
+
+name := "geotrellis-geotools"
+
+libraryDependencies ++= Seq(
+  "org.geotools" % "gt-main" % Version.geotools,
+  "org.geotools" % "gt-coverage" % Version.geotools,
+  "org.geotools" % "gt-geotiff" % Version.geotools,
+  "org.geotools" % "gt-epsg-hsql" % Version.geotools,
+  "org.apache.spark" %% "spark-core" % Version.spark % "provided",
```

Comment:
Indeed, the first few commits of this PR were taken from my other GeoTools-related PR.  When the `GridCoverage2D` stuff that Rob is working on goes in, I will rebase and this will go away.